### DISCUSSION TITLE
failed: Connection timed out.

### DISCUSSION THREAD

**Discussion by AmitvikramPujar** [ohmyzsh/ohmyzsh#11908] [Help] [answered]
Dear **`ohmyzsh`** community,

I have installed zsh (v5.9) on Ubuntu 23.04(Arch=x86_64).
However, when I try installing **ohmyzsh** through `curl` or `wget` as follows, it just takes too long to connect to the URL: 
```shell
sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
``` 
OR
```shell
sh -c "$(wget -O- https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
```
The message I get is: 
`> failed: Connection timed out.
` 
I have tried multiple times even after resetting my network connection, but to no avail. Please suggest improvements or some alternatives that I can implement to install **`ohmyzsh`**.

PS: I have installed zsh from the apt repositor using the following command:
```shell
sudo apt-get install zsh
``` 
However, when I look for .zshrc file, it is found at `/etc/zsh/.zshrc` & not at `$HOME` as mentioned in the README.

Thanks!

**Comment by carlosala** [comment, marked as answer]
Try to use our mirror https://install.ohmyz.sh
